bench_speedups.py

```diff
@@ -0,0 +1,11 @@
+from markupsafe._speedups import escape
+
+for _name, s in (
+    ("short escape", '"<strong>Hello, World!</strong>"'),
+    ("long escape", '"Hello, World!" * 1000'),
+    ("short plain", '"Hello, World!"'),
+    ("long plain", '"Hello, World!" * 1000'),
+    ("long suffix", '"<strong>Hello, World!</strong>" + "x" * 100_000'),
+):
+    for _ in range(1_000_000):
```

Comment:
with micro benchmarks I find that calling the function 1000's of times in a loop helps reduce the noise. This extra file isn't needed but it's what I used to measure the difference